lib/app/features/chat/messages/views/pages/messages_page.dart

```diff
@@ -45,7 +45,8 @@ class MessagesPage extends ConsumerWidget {
                           );
                         case 1:
                           return const TextMessage(
-                            message: 'How are you doing?',
+                            message:
+                                'How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing?How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing? How are you doing?',
```

Comment:
Nerd comment 🤓
Maybe it is better to use [lorem ipsum](https://loremipsum.io/generator?n=5&t=p) text and marked that this is mock some how.

Reply:
It looks more good 🌟  thanks for suggestion @ice-alcides !